mfes/registration/src/app/otp/page.tsx

```
@@ -0,0 +1,180 @@
+      }
+
+      onChange(otpArray.join(''));
+    }
+  };
+
+  return (
+    <Box sx={{ display: 'flex', gap: 1, alignItems: 'center' }}>
+      {new Array(length).fill(null).map((_, index) => (
+        <React.Fragment key={index}>
+          <TextField
+            inputRef={(el) => (inputRefs.current[index] = el!)}
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)